### PR TITLE
Guide for troubleshooting Agent with Cloud connection for new nodes

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

Added a guide to help users with newly claimed nodes that are not connecting to Cloud

##### Test Plan

n/a

##### Additional Information
This addresses netdata/learn#1080

### CODE REVIEW

File: docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+<!--
+title: "Troubleshooting Agent with Cloud connection"
+description: "A simple guide to troubleshoot occurances where the Agent is showing as offline after claiming."
+custom_edit_url: https://github.com/netdata/netdata/edit/master/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md
+-->
+
+# Troubleshooting Agent with Cloud connection
+
+Sometimes, when claiming a node, it might not show up as online in Netdata Cloud.  
+The occurances triggering this behavior might be:
```
Comment (shyamvalsan) [inline docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md:10]: occurances --> occurrences

File: docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+- [The claiming script failed](#the-claiming-script-failed)
+- [Claiming on an older, deprecated version of the Agent](#claiming-on-an-older-deprecated-version-of-the-agent)
+- [Network issues while connecting to the Cloud](#network-issues-while-connecting-to-the-cloud)
```
Comment (shyamvalsan) [inline docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md:14]: An additional one we could add here is the case where agent couldn't be reached and needs a restart after install/claim

Comment (shyamvalsan) [reply docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md:14]: in this case the claiming script doesn't necessarily say it failed. But in the logs there will be a statement that agent needs to be restarted, could be hard to spot for users. So good to call it out I think.

File: docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+Read more about [Starting, Stopping and Restarting the Agent](https://learn.netdata.cloud/docs/configure/start-stop-restart).
+
+:::
+
+## Claiming on an older, deprecated version of the Agent
+
+Make sure that you are using the latest version of Netdata if you are using the [Claiming script](https://learn.netdata.cloud/docs/agent/claim#claiming-script).
+
+With the introduction of our new architecture, Agents running versions lower than `v1.32.0` can face claiming problems, so we reccomend to [update the Netdata Agent](https://learn.netdata.cloud/docs/agent/packaging/installer/update).
```
Comment (shyamvalsan) [inline docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md:60]: reccomend -> recommend

Comment (shyamvalsan) [reply docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md:60]: maybe also note that the recommendation is to be on the latest netdata version, in any case they should be on latest stable.

File: docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+main-ingress-545609a41fcaf5d6.elb.us-east-1.amazonaws.com has address 44.196.50.41
+```
+
+If the command fails, you can whitelist the `app.netdata.cloud` domain from the node's firewall restrictions.
+
+:::tip
+
+If you can't whitelist domains from your firewall, you can whitelist the IPs that the above mentioned command will produce, but keep in mind that they can change at anytime without notice.
+
+:::
```
Comment (shyamvalsan) [inline docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md:128]: I think the certificate issue might be another one worth noting here - I've seen multiple users run into this


https://community.netdata.cloud/t/certificate-verification-error-connecting-to-the-cloud/1790

 https://community.netdata.cloud/t/claimed-nodes-not-reporting-to-cloud/1282/7

File: docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+sudo systemctl start netdata 
+```
+
+:::info
+
+Read more about [Starting, Stopping and Restarting the Agent](https://learn.netdata.cloud/docs/configure/start-stop-restart).
+
+:::
+
+## Claiming on an older, deprecated version of the Agent
```
Comment (shyamvalsan) [inline docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md:56]: One reason why users may get an older version unknowingly is if they are doing package installs and the package manager repo (for whatever reason) contains an older version OR if they are trying to claim a node on an unsupported (by Netdata) OS version such as Ubuntu 16.04. 

In such cases I believe the recommendation is to do a static build install 
https://learn.netdata.cloud/docs/agent/packaging/installer/methods/kickstart#static-builds

File: docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md (hunk 4 above)
Comment (shyamvalsan) [reply docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md:128]: We should add a section at the end which says if none of these steps worked then the user should reach out to us via discord/forum/github etc.

Comment (shyamvalsan) [reply docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md:128]: And finally, we should let the SREs do a quick review of this as well to ensure it's technically accurate.

**Review by shyamvalsan — CHANGES_REQUESTED** [review]

File: docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md (hunk 2 above)
Comment (Ancairon) [reply docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md:14]: Will replace "failed" with "unsuccessful", and then before pointing out "if the Agent is not running" put a tip that the solution might be to restart it

**Review by shyamvalsan — DISMISSED** [review]

File: docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md (hunk 4 above)
Comment (DShreve2) [reply docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md]: Agree on the discord/forum/github guidance!

**Review by DShreve2 — APPROVED** [review]

File: docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md (hunk 4 above)
Comment (DShreve2) [reply docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md]: We can tag the SRE team, @shyamvalsan , @Ancairon to do a quick read through of this.

**Review by shyamvalsan — APPROVED** [review]

**Comment by Ancairon** [comment]
This looks good, merging.

**Pull request merged**